### PR TITLE
document LC data serving options

### PR DESCRIPTION
**Pull request by etan-status**
Adds a documentation page for configuring LC data serving.

### CODE REVIEW

File: docs/the_nimbus_book/src/light-client-data.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Light client data (advanced)
+
+Nimbus is configured by default to serve data that allows light clients to stay in sync with the Ethereum network. Light client data is imported incrementally and does not affect validator performance. Information about the light client sync protocol can be found in the [Ethereum consensus specs](https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/light-client/sync-protocol.md).
+
+!!! note
+    Nimbus also implements a [standalone light client](./el-light-client.md) that may be used to sync an execution layer (EL).
```
Comment (tersec) [inline docs/the_nimbus_book/src/light-client-data.md]: Nimbus documentation, I think, has referred to specific software/clients providing the EL interface as an execution client (or, execution layer client).

File: docs/the_nimbus_book/src/options.md [hunk 2]
```diff
@@ -34,20 +34,23 @@ The following options are available:
...
+     --web3-url                One or more execution layer Web3 provider URLs.
      --non-interactive         Do not display interative prompts. Quit on missing configuration.
      --netkey-file             Source of network (secp256k1) private key file (random|<path>) [=random].
      --insecure-netkey-password  Use pre-generated INSECURE password for network private key file [=false].
      --agent-string            Node agent string which is used as identifier in network [=nimbus].
      --subscribe-all-subnets   Subscribe to all subnet topics when gossiping [=false].
      --num-threads             Number of worker threads ("0" = use as many threads as there are CPU cores
                                available) [=0].
+     --jwt-secret              A file containing the hex-encoded 256 bit secret key to be used for
+                               verifying/generating jwt tokens.
```
Comment (tersec) [inline docs/the_nimbus_book/src/options.md]:
```suggestion
                               verifying/generating JWT tokens.
```

File: docs/the_nimbus_book/src/light-client-data.md (hunk 1 above)
Comment (etan-status) [reply docs/the_nimbus_book/src/light-client-data.md]: 0189c555b1f3a8623d8f0f8b3d44f2c67bf70628

File: docs/the_nimbus_book/src/options.md (hunk 2 above)
Comment (etan-status) [reply docs/the_nimbus_book/src/options.md]: b6ecc91c58eb0ca34e51c7c4da60459d22d89341

**Pull request merged**